webroot/results/admin/export_public.php

```diff
@@ -36,16 +36,17 @@
     # wcaDelegate and organiser fields by joining with the users,
     # competition_delegates, and competition_organizers tables.
     'Competitions' => 'SELECT Competitions.id, Competitions.name, Competitions.cityName, Competitions.countryId, Competitions.information, Competitions.year,
-                              Competitions.month, Competitions.day, Competitions.endMonth, Competitions.endDay, Competitions.eventSpecs,
+                              Competitions.month, Competitions.day, Competitions.endMonth, Competitions.endDay, replace(GROUP_CONCAT(competition_events.event_id), ",", " ") as eventSpecs,
                               GROUP_CONCAT(DISTINCT(CONCAT("[{", users_delegates.name, "}{mailto:", users_delegates.email, "}]")) SEPARATOR " ") as wcaDelegate,
                               GROUP_CONCAT(DISTINCT(CONCAT("[{", users_organizers.name, "}{mailto:", users_organizers.email, "}]")) SEPARATOR " ") as organiser,
                               Competitions.venue, Competitions.venueAddress,
                               Competitions.venueDetails, Competitions.external_website, Competitions.cellName, Competitions.latitude, Competitions.longitude
                               FROM Competitions
+                              LEFT JOIN competition_events ON Competitions.id = competition_events.competition_id
                               LEFT JOIN competition_delegates ON Competitions.id=competition_delegates.competition_id LEFT JOIN users AS users_delegates ON users_delegates.id=competition_delegates.delegate_id
                               LEFT JOIN competition_organizers ON Competitions.id=competition_organizers.competition_id LEFT JOIN users AS users_organizers ON users_organizers.id=competition_organizers.organizer_id
                               WHERE Competitions.showAtAll=1
-                              GROUP BY competition_delegates.competition_id',
+                              GROUP BY Competitions.id',
```

Comment:
Yes, just cleaning up. Good call to run the query. I needed to change it slightly. Now the queries are the same except for the order of the eventspecs.

[export_orig.txt](https://github.com/cubing/worldcubeassociation.org/files/454916/export_orig.txt)

[export_new.txt](https://github.com/cubing/worldcubeassociation.org/files/454917/export_new.txt)